src/composable/useSearchDeveloper.js

```diff
@@ -0,0 +1,31 @@
+import sortBy from "lodash/sortBy";
+import Searcher from "@/util/Searcher";
+import store from "../store/store";
+
+const developerSearcher = new Searcher().setField("name").setField("login");
+
+export function useSearchDeveloper() {
+  function searchPopular(searchTerm) {
+    let developers = store.state.Developers.developers;
+    developerSearcher.setData(developers);
+
+    if (searchTerm) {
+      developers = developerSearcher.findAll(searchTerm);
+    }
+
+    return developers.slice(0, 10);
+  }
+
+  function searchNewDevelopers(searchTerm) {
+    let developers = store.getters["Developers/newDevelopers"];
+    developerSearcher.setData(developers);
+
+    if (!searchTerm) {
+      developers = developers.slice(0, 10);
+    }
+
+    return sortBy(developerSearcher.findAll(searchTerm), dev => -Number(new Date(dev.createdAt))).slice(0, 10);
```

Comment:
I missed this @DannyFeliz, I think we have the same situation as before, the search is being executed regardless of whether there is a search tearm or not